eng/dockerfile-templates/aspnet/6.0/Dockerfile.mariner

```
@@ -0,0 +1,15 @@
+ARG REPO=mcr.microsoft.com/dotnet/runtime
+FROM $REPO:6.0-{{OS_VERSION}}{{ARCH_TAG_SUFFIX}}
+
+ENV \
+    # ASP.NET Core version
+    ASPNET_VERSION={{VARIABLES["aspnet|6.0|build-version"]}} \
+    # Set the default console formatter to JSON
+    Logging__Console__FormatterName=Json
+
+# Install ASP.NET Core
+RUN curl -SL --output aspnetcore.rpm https://dotnetcli.azureedge.net/dotnet/aspnetcore/Runtime/$ASPNET_VERSION/aspnetcore-runtime-$ASPNET_VERSION-{{ARCH_SHORT}}.rpm \
+    && dotnet_sha512='{{VARIABLES[cat("aspnet-rpm|6.0|linux|", ARCH_SHORT, "|sha")]}}' \
+    && echo "$dotnet_sha512 aspnetcore.rpm" | sha512sum -c - \
```

```suggestion
    && echo "$dotnet_sha512  aspnetcore.rpm" | sha512sum -c - \
```

IIRC the two spaces are critical here.  We should be consistent across our Dockerfiles.  If it's not needed we should clean it up.

---

I see that Dockerfile.linux uses one space while Dockerfile.alpine uses two.  I'll log an issue to address this.